ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -40,56 +29,61 @@ public final class AliasTokens {
         case brandForeground5
         case brandForegroundInverted
         case brandForegroundDisabled
-
+        // Background colors
+        case background1
+        case background1Pressed
```

Comment:
The [ButtonDynamicColors](https://github.com/microsoft/fluentui-apple/blob/a65fb6224a789f562ca51c5f12b264b2f4a37f97/ios/FluentUI/Core/Theme/Tokens/ButtonDynamicColors.swift#L7) has a structured way to define different colors for different states.
This enumeration is just concatenating the states to the type of color.

Can we have a way to reuse that structure or a similar idea so we don't have a combination matrix of all **colors x states**?

Reply:
See response to conversation - organizing it in the style of ButtonDynamicColors would defeat purposes 1 and 2, and I believe add unnecessary complication to what should be a very straightforward set of enums.